jenkins/job-configs/kubernetes-jenkins/bootstrap-ci.yaml

```diff
@@ -120,12 +120,6 @@
     # END AWS
 
     # gce-manual-jobs #gmarek
-    - kubernetes-e2e-gce-enormous-cluster:
```

Comment:
It'd be replaced with gce-5000-performance job. That's part of https://github.com/kubernetes/test-infra/pull/3456.